src/core/ComputeTs.c

```diff
@@ -61,7 +108,7 @@ void _ComputeTs(int snapshot)
 #endif
 
   // TODO: Can we reduce the scope of these variables and, if not, improve the names?
-  double weight = 0;
+  double weight, total_weight;
```

Comment:
Extremely minor thing, isn't it better to initialize these two variables to zero?

Reply:
maybe to 1?

Reply:
weight is initialized to 1 under and total_weight doesnt seem to matter